users/tests/test_unit_validators.py

```diff
@@ -81,3 +83,19 @@ def test_validator_should_pass_when_name_contain_dash(self):
             self.user_name_validator("Kowalska-Nowak")
         except ValidationError:
             self.fail()
+
+
+class TestReusedPreviousPasswordValidator(TestCase):
+    def setUp(self) -> None:
+        self.password_validator = ReusedPreviousPasswordValidator()
+        self.user = UserFactory()
+        self.password = "Password1234!"
+        self.user.set_password(self.password)
+        self.user.save()
+
+    def test_that_validator_raise_exception_when_new_password_is_the_same_as_previous_password(self):
+        with self.assertRaises(ValidationError) as exception:
+            self.password_validator.validate(password=self.password, user=self.user)
+        self.assertEqual(
+            exception.exception.messages, [ValidationErrorText.VALIDATION_ERROR_NEW_PASSWORD_THE_SAME_AS_PREVIOUS]
+        )
```

Comment:
You should have at least one test that checks if different password pass as assumed.

Reply:
Done